src/parsers/perf/perfparser.cpp

```diff
@@ -215,20 +215,24 @@ QDebug operator<<(QDebug stream, const LocationDefinition& locationDefinition)
 struct Symbol
 {
     StringId name;
+    quint64 relAddr;
```

Comment:
both = 0

Reply:
Sorry, missed the remark, I will add into next iteration of review